<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftPanel</class>
 <widget class="QWidget" name="LeftPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="leftSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="TaxListWidget" name="taxList" native="true"/>
     <widget class="CurrentTaxNodeDetails" name="curNode" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaxListWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui_components/taxlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CurrentTaxNodeDetails</class>
   <extends>QWidget</extends>
   <header location="global">ui_components/currenttaxnodedetails.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
